internal/nri/nri.go

```diff
@@ -169,6 +172,25 @@ func (l *local) RunPodSandbox(ctx context.Context, pod PodSandbox) error {
 	return err
 }
 
+func (l *local) UpdatePodSandbox(ctx context.Context, pod PodSandbox) error {
+	if !l.IsEnabled() {
+		return nil
+	}
+
+	l.Lock()
+	defer l.Unlock()
+
+	request := &nri.UpdatePodSandboxRequest{
+		Pod:                    podSandboxToNRI(pod),
+		OverheadLinuxResources: &nri.LinuxResources{},
+		LinuxResources:         &nri.LinuxResources{},
```

Comment:
How can populate &nri.LinuxResources?

Reply:
you can add a function to sandbox_linux.go that populates a created UPdatePodSandboxRequest object with the {Overhead,}LinuxResources objects.

Reply:
nice... like [this](https://github.com/cri-o/cri-o/pull/9016/files#diff-b106070a06d88855804587b5d63197c9ebe5f8fdd5dff62abd1e697b46893db6R24)?